website/screens/common/pagesList.ts

```diff
@@ -25,6 +25,7 @@ export const themeGeneratorLinks = [
 const overviewLinks: LinkDetails[] = [
   { label: "Introduction", path: "/overview/introduction", status: "ready" },
   { label: "Releases", path: "/overview/releases", status: "ready" },
+  { label: "Component lifecycle", path: "/overview/component-lifecycle", status: "ready" },
 ];
```

Comment:
I would put `Component lifecycle` page before `Releases` in the Sidenav, to follow the alphabetical order.